docs/root/configuration/http/http_filters/aws_request_signing_filter.rst

```diff
@@ -74,6 +74,24 @@ An example of configuring this filter to use ``AWS_SIGV4A`` signing with a wildc
 
 .. include:: _include/aws_credentials.rst
 
+Configuration as an upstream HTTP filter
+----------------------------------------
+SigV4 or SigV4A request signatures cover the HTTP host, URL and payload. Depending on the configuration, Envoy may modify these during
+prior to forwarding to the Cluster subsystem, but after the signature has been calculated and inserted into the HTTP headers. This will
+invalidate the signature, by design.
+To resolve this situation, the AWS Request Signing Filter can be configured as an upstream HTTP filter. This allows signatures to be
+calculated as a final step before the HTTP request is forwarded upstream, ensuring signatures are correctly calculated over the updated
+HTTP fields.
+
+An example of configuring this filter as an upstream HTTP filter:
+
+.. literalinclude:: _include/aws-request-signing-filter-upstream.yaml
+    :language: yaml
+    :lines: 56-100
```

Comment:
this is quite a lot of config - could you highlight the lines that are directly required for setup

Reply:
as the config is downloadable you dont need to show any more context than 2 lines before and 2 lines after the specific configuration. 

perhaps add a comment to the effect of "upstreams can be configured in a similar way, for example ..."

relatedly im wondering if all of the config is actually necessary (in all of the example files) - eg DNS setup - if it is necessary then i think this needs an explanation, if not i think it needs to be removed

Reply:
have reduced the config file provided to the minimum and adjusted as per suggestion.